### PR TITLE
[4.1] Fix for incorrect path name breadcrumbs

### PR DESCRIPTION
**Pull request by sandewt**
Code changes module breadsrumbs

Thanks @pippo8765

Pull Request for Issue #37076.

### CODE REVIEW

**Comment by richard67** [comment]
@sandewt Maybe you can change the title of this PR to something more suitable? Currently it reads like a proposal for an improvement, but in fact it's a bug fix. A better title will help to find human testers, I think. Thanks in advance.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 155ba25227cf3fe3ae53987283f1a04bb3bcb6b0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37282">issues.joomla.org/tracker/joomla-cms/37282</a>.</sub>

File: modules/mod_breadcrumbs/tmpl/default.php [hunk 1]
```diff
@@ -57,14 +57,15 @@
...
 					$breadcrumbItem = '<span>' . $item->name . '</span>';
 				endif;
+				echo '<li class="mod-breadcrumbs__item breadcrumb-item' . $class . '">' . $breadcrumbItem . '</li>';
 
 			elseif ($show_last) :
 				// Render last item if required.
 				$breadcrumbItem = '<span>' . $item->name . '</span>';
 				$class          = ' active';
+				echo '<li class="mod-breadcrumbs__item breadcrumb-item' . $class . '">' . $breadcrumbItem . '</li>';
 			endif;
```
Comment (Quy) [inline modules/mod_breadcrumbs/tmpl/default.php]:
```suggestion
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 155ba25227cf3fe3ae53987283f1a04bb3bcb6b0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37282">issues.joomla.org/tracker/joomla-cms/37282</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37282">issues.joomla.org/tracker/joomla-cms/37282</a>.</sub>

**Comment by richard67** [comment]
Hint for the one who merges: If possible, edit the commit description and add @pippo8765 as co-author.

File: modules/mod_breadcrumbs/tmpl/default.php (hunk 1 above)
Comment (sandewt) [reply modules/mod_breadcrumbs/tmpl/default.php]: 👍

**Comment by pippo8765** [comment]
Thnx!
  

  

  
  
 -- S poštovanjem,/Kind regards, Filip Jovic M.Sc. E. E. JUNIS, University of Niš +381 18 257 085, lok: 152 st: Univezitetski trg 2 ------------------------------------------- +381 64 61 54 557 ***@***.***  

  
  
  
  
  
>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**